### PR TITLE
add Places Sorter inner component - interface and random implementation

### PR DESCRIPTION
**Pull request by karenav**

### CODE REVIEW

File: src/main/webapp/java/data/RandomPlacesSorter.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import java.util.List;
+import java.util.Collections;
+
+class RandomPlacesSorter implements PlacesSorter {
+
+    public static List<Place> sortPlaces(List<Place> unordered) {
+        Collections.shuffle(unordered);
+        return unordered;
```
Comment (tbarnahor) [inline src/main/webapp/java/data/RandomPlacesSorter.java:8]: I am wondering if we should return here only the three final places, what are do we get by returning the whole list?

File: src/main/webapp/java/data/PlacesSorter.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import java.util.List;
+ 
+interface PlacesSorter {
```
Comment (tbarnahor) [inline src/main/webapp/java/data/PlacesSorter.java:3]: Looks good. Just not sure if the name "unordered" param name is clear enough.

File: src/main/webapp/java/data/RandomPlacesSorter.java (hunk 1 above)
Comment (karenav) [reply src/main/webapp/java/data/RandomPlacesSorter.java:8]: We get more flexability, our applications takes the top 3 but the backend supports more than that (following what Ariel said)

File: src/main/webapp/java/data/PlacesSorter.java (hunk 2 above)
Comment (karenav) [reply src/main/webapp/java/data/PlacesSorter.java:3]: sure np. done

File: src/main/webapp/java/data/RandomPlacesSorter.java (hunk 1 above)
Comment (tbarnahor) [reply src/main/webapp/java/data/RandomPlacesSorter.java]: Ok makes sense for now.

**Pull request merged**